### ISSUE TITLE
QUESTION what to do after beating the lvl 40 Druddigon in the palace

### ISSUE THREAD

**Issue by microxvzla** [fuji97/pokemon-xenoverse-tracker#175]

**Comment by MoroUC** [comment]
Druddigon is the first trial (the fire trial), after that you have to achieve second (the water one) and third trial (grass), then you can fight Rochelle.

**Comment by microxvzla** [comment]
yeah but it doesnt let me fight again, where do i have to go to beat the other trial?

**Comment by Reol1000** [comment]
You have to go in the same place again

**Comment by microxvzla** [comment]
i don't know if it is a bug or something, the guard doesn't let me in again and i can't seem to be able to capture pokemon now, it says i cant capture the pokemon :(

**Comment by KetsubanZero** [comment]
you need to have the old rod to do the second trial, you can obtain it by a fisherman in Bosco Mela, near to a lake (you should complete a minigame)
